doc/reference/instance_options.md

```diff
@@ -372,6 +372,7 @@ The following instance options control the {ref}`security` policies of the insta
 
 Key                                             | Type      | Default           | Live update   | Condition                 | Description
 :--                                             | :---      | :------           | :----------   | :----------               | :----------
+`security.csm`                                  | bool      | `false`           | no            | virtual machine           | Controls whether to use a firmware that supports UEFI-incompatible operating systems
```

Comment:
Is it worth adding in the docs that this functionality is dependent on a custom built firmware only available in the snap package?

Reply:
I don't think so as other distros do have CSM builds, ArchLinux at least does.

Reply:
And it uses the same file naming patterns?